upi/vsphere/stage2/9.post-deployment/playbooks/infra.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- hosts: localhost 
+  tasks:
+  - name: Create default nodeSelector to target workers
+    k8s:
+      state: present
+      definition:
+        apiVersion: config.openshift.io/v1
+        kind: Scheduler
+        metadata:
+          name: cluster
+        spec:
+          defaultNodeSelector: node-role.kubernetes.io/worker=""
```

Comment:
This is literally setting the default node selector to two double-quotes, so nothing wants to schedule because it is looking for

node-role.kubernetes.io/infra: ""

So pods don't schedule with errors like:

"Error creating build pod: unable to parse requirement: label value: Invalid value: """": must match format [ DNS 1123 subdomain / ] DNS 1123 label"

Reply:
Probably just wants to be `node-role.kubernetes.io/worker=`